src/components/availability-panel/helper-functions/AssignTask.ts

```diff
@@ -0,0 +1,31 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+type Props = {
+  authToken: string;
+  taskId: string;
+  assignee: string;
+};
+
+const AssignTask = async ({ authToken, taskId, assignee }: Props) => {
+  try {
+    const data = JSON.stringify({
+      status: 'assgined',
+      assignee,
+    });
+    const config: AxiosRequestConfig = {
+      method: 'patch',
+      url: `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/${taskId}`,
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${authToken}`,
+      },
+      data,
+    };
+    const response = await axios(config);
+    return response;
+  } catch (err) {
+    return err;
+  }
+};
```

Comment:
No need for the `authToken` argument,
We use `Bearer token` in Staging environment and for the production environment, we use Cookies which doesn't need to be manually added, Cookies automatically go with every request.